Networking/Networking/Remote/AccountRemote.swift

```diff
@@ -17,6 +17,22 @@ public class AccountRemote: Remote {
     }
 
 
+    /// Loads the AccountSettings associated with the Credential's authToken.
```

Comment:
I am not sure if this is relevant, but other methods that receive a `userID` as parameter as annotated as `Loads ... with the WordPress.com user`

Reply:
Technically the user ID is ignored in this case but we're passing it along for the ride to make it into the Storage layer. We're relying upon the bearer token to provide the magic on the API side to return the settings for the user.